src/Anonymity/BulkRound.cpp

```
@@ -64,6 +64,10 @@
 
   void Xor(QByteArray &dst, const QByteArray &t1, const QByteArray &t2)
   {
+    if(t1.count() != t2.count()) {
```

I just added a comment in the docstring for the method. Since XORing two bitstrings of unequal length is always an error (right?) I think we should fail loudly when it happens. If not, we could add a return value to the function and check it every time.